vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/embedding_pipeline_value_util.cc

```diff
@@ -49,8 +49,8 @@ absl::optional<EmbeddingPipelineInfo> EmbeddingPipelineFromValue(
   }
 
   embedding_pipeline.dimension = 1;
-  for (const auto [key, value] : *value) {
-    const auto* list = value.GetIfList();
+  for (const auto [k, v] : *value) {
```

Comment:
Can we please revert as we should not use abbreviations. Thanks

Reply:
We do need to rename this as value shadows a variable, but I renamed to `embedding_key` and `embedding_value` in b4174716b278ed73d7aab2c1ed841d54e25cb51a.